### DISCUSSION TITLE
Question regarding "Warning: You have specified that the 'wait' date is after the 'due' date."

### DISCUSSION THREAD

**Discussion by kwoot** [GothenburgBitFactory/taskwarrior#2450] [Q&A] [answered]
This came up when I bulk moved a set of task with the same tag to the future.

Is there some query I can run to see which task has a wait date after the due date?

**Comment by tbabej** [comment, marked as answer]
To list all the tasks that have a wait date after the due date, you can use the following command: `task wait.after:due all`

**Comment by kwoot** [comment]
Excellent! Works like a charm. Downside: got some home work to do :-)
